src/api/java/com/minecolonies/api/crafting/RecipeStorage.java

```diff
@@ -365,4 +491,65 @@ private void insertCraftedItems(final List<IItemHandler> handlers)
             }
         }
     }
+
+    @Override
+    public RecipeStorage getClassicForMultiOutput(final ItemStack requiredOutput)
+    {
+        return StandardFactoryController.getInstance().getNewInstance(
```

Comment:
You are in the concrete implementation, you do not need to use the factory controller......
You can invoke yourself just fine,

Reply:
Yeah, cleaning this up